content/en/docs/reference/access-authn-authz/certificate-signing-requests.md

```diff
@@ -272,7 +279,6 @@ kubectl create rolebinding developer-binding-john --role=developer --user=john
 ### Add to kubeconfig
 
 The last step is to add this user into the kubeconfig file.
-This example assumes the key and certificate files are located at "/home/vagrant/work/".
 
 First, you need to add new credentials:
 
```

Comment:
can change the invocation to `kubectl config set-credentials john --client-key=john.key --client-certificate=john.crt --embed-certs=true` and drop assumptions about absolute paths

Reply:
done